infra/build/developer-tools/build/scripts/task_wrapper_scripts/terraform_validate

```diff
@@ -19,8 +19,5 @@
 # files to validate.  This directory is assumed to be a root module.
 
 set -eu
```

Comment:
@apeabody do you think setting `o pipefail` is a good idea here too?

Reply:
Hmm, this can also be called via parallel (used by https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/Makefile#L72), so we should just be careful that use case doesn't cause a probleme: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/build/developer-tools/build/scripts/task_helper_functions.sh#L194

Reply:
since there are no pipes, it probably doesn't matter too much? I usually try to use it as a matter of habit, but probably can leave it out here.

Reply:
Yeah, it should be fine.  If you have done a tested against a parallel linted repo, that's good enough for me.